autotyping/guess_type.py

```diff
@@ -40,15 +40,16 @@ def guess_type_from_argname(name: str) -> Tuple[Optional[str], List[str]]:
         # only do a simple container match
         # and don't check all of BOOL_NAMES to not trigger on stuff like "save_list"
         elems = m.group("elems")
-        for names, name_type in (
```

Comment:
As mentioned in the issue, let's fix this instead by using `+` instead of `*` on the two regexes above this if.